content/client-lib-development-guide/features.textile

```diff
@@ -1534,40 +1478,59 @@ enum StatsIntervalGranularity:
   DAY
   MONTH
 
+class DeviceDetails:
+  + getDeviceInfo() => io DeviceDetails
+  id: String
+  platform: DeviceType
+  type: DeviceType
+  clientId: String?
+  metadata: Dict<String, String>
+  updateTokenId: String
+  resetId(String)
+  resetUpdateToken() => io
```

Comment:
Ok, sure, so remove any ability for the device to automatically renew the token and only allow it to be set explicitly with `reset`?